stage1/stop/stop.mk

```diff
@@ -0,0 +1,13 @@
+STP_FLAVORS := $(filter-out kvm,$(STAGE1_FLAVORS))
+
+ifneq ($(STP_FLAVORS),)
+
+ASSCB_FLAVORS := $(STP_FLAVORS)
```

Comment:
This should be ASGB (from Aci Simple Go Binary, which you include below) prefix, not ASSCB (which is from Aci Simple Static C Binary).